src/api/v1/chain.ts

```diff
@@ -22,6 +29,29 @@ export class ChainAPI {
         })
     }
 
+    async get_block(block_num_or_id: string | number | UInt32) {
+        return this.client.call({
+            path: '/v1/chain/get_block',
+            params: {block_num_or_id},
+            responseType: GetBlockResponse,
+        })
+    }
+
+    async get_currency_balance(contract: NameType, accountName: NameType, symbol?: NameType) {
+        const params: any = {
+            account: Name.from(accountName),
+            code: Name.from(contract),
+        }
+        if (symbol) {
+            params.symbol = symbol
```

Comment:
Missing a `Name.from` here

Reply:
And is symbol really sent as a Name to the api?

Reply:
Hrm, doesn't look like it actually is. I thought I had found that in the code somewhere, but the API documentation states:

```string (Symbol) ^([0-9]{1,32}.[0-9]{4} [A-Z]{1,7})$```

So apparently it has its own type. Maybe this is best just as a string?